### PR TITLE
feat(test):[TRI-292] add postman collection for IRS API tests

### PR DESCRIPTION
**Pull request by mkanal**

### CODE REVIEW

File: testing/IRS DEMO Collection.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+							"key": "jobStates",
+							"value": "COMPLETED"
+						}
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "2.0.1 [Search for Jobs in error state]  http://irs.int.demo.catena-x.net/irs/jobs?jobStates=ERROR",
```
Comment (ds-jkreutzfeld) [inline testing/IRS DEMO Collection.postman_collection.json:110]: The description does not match the actual call.

File: testing/IRS DEMO Collection.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+							"key": "jobStates",
+							"value": "COMPLETED"
+						}
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "2.0.2 [Search for Jobs in error state]  http://irs.int.demo.catena-x.net/irs/jobs?jobStates=INITIAL",
```
Comment (ds-jkreutzfeld) [inline testing/IRS DEMO Collection.postman_collection.json:145]: The description does not match the actual call.

File: testing/IRS DEMO Collection.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+							"key": "jobStates",
+							"value": "COMPLETED"
+						}
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "2.1.0 [Search for error Jobs]  http://irs.int.demo.catena-x.net/irs/jobs?jobStates=COMPLETED",
```
Comment (ds-jkreutzfeld) [inline testing/IRS DEMO Collection.postman_collection.json:215]: The description does not match the actual call.

File: testing/IRS DEMO Collection.postman_collection.json [hunk 4]
```diff
@@ -0,0 +1,342 @@
...
+				"body": {
+					"mode": "formdata",
+					"formdata": [
+						{
+							"key": "",
+							"value": "9c2f591a-9779-4cb6-81f9-bb3b219da30a",
+							"type": "text"
+						}
+					]
+				},
```
Comment (ds-jkreutzfeld) [inline testing/IRS DEMO Collection.postman_collection.json:279]: Why the body for a GET?

File: testing/IRS DEMO Collection.postman_collection.json [hunk 5]
```diff
@@ -0,0 +1,342 @@
...
+						"warning": "This is a duplicate header and will be overridden by the Content-Type header generated by Postman.",
+						"key": "Content-Type",
+						"value": "application/json",
+						"type": "text"
+					},
+					{
+						"key": "jobId",
+						"value": "9c2f591a-9779-4cb6-81f9-bb3b219da30a",
+						"type": "text",
+						"disabled": true
```
Comment (ds-jkreutzfeld) [inline testing/IRS DEMO Collection.postman_collection.json:267]: This header is not needed

File: testing/IRS DEMO Collection.postman_collection.json [hunk 6]
```diff
@@ -0,0 +1,342 @@
...
+							"value": "",
+							"disabled": true
+						}
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "4.0.0 [Search for error Jobs]  http://irs.int.demo.catena-x.net/irs/jobs?jobStates=ERROR",
```
Comment (ds-jkreutzfeld) [inline testing/IRS DEMO Collection.postman_collection.json:307]: The description does not match the actual call.

File: testing/IRS DEMO Collection.postman_collection.json (hunk 1 above)
Comment (mkanal) [reply testing/IRS DEMO Collection.postman_collection.json:110]: ?

**Review by ds-jkreutzfeld — APPROVED** [review]

**Pull request merged**